src/components/FinalIntakeScreen/FinalIntakeScreen.tsx

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React from 'react';
+import { PageContainer } from 'screens/styles';
+import HomeScreen from 'screens/Home/HomeScreen';
+import UploadScreen from 'screens/Documents/Documents';
+import { TextRegular, TextBold, TextRegularWhite } from 'assets/fonts/Fonts';
+import { ButtonDark, ButtonLight } from 'assets/Components';
+import {
+  ContentContainer,
+  TitleContainer,
+  ButtonContainer,
+  ButtonView,
+} from './styles';
+
+const FinalIntakeScreen = () => {
+  return (
+    <PageContainer>
+      <ContentContainer>
+        <TitleContainer>
+          <TextBold>Your intake form has been successfully submitted!</TextBold>
+        </TitleContainer>
+        <TextRegular>
+          Please go upload your legal documents needed to properly assess your
+          case. If you do not have your documents on hand at this time...
+        </TextRegular>
+      </ContentContainer>
+      <ButtonContainer>
+        <ButtonView>
+          <ButtonLight
+            onPress={() => {
+              <HomeScreen />;
```

Comment:
We might have to use the navigation stack to properly change screens here. I tried something like this: 

`<ButtonLight onPress={() => navigation.navigate('Home', { screen: 'Forms' })} >`

but got an undefined screen error. This might require refactoring parts of `RootNavigator` to accommodate this. I'm going to move this to another PR, since this seems more involved than anticipated, but let me know if you stumble upon a solution! 

https://stackoverflow.com/questions/63566372/how-do-navigate-to-a-tab-from-another-tab-using-react-navigation-v5